.github/workflows/test_child.yml

```
@@ -133,6 +133,38 @@ jobs:
           path: rvc4_test_output.log
           if-no-files-found: warn
 
+ linux_rvc4lite_test:
+      needs: [build_docker_container]
+      strategy:
+        matrix:
+          rvc4os: ${{ fromJson(inputs.luxonis_os_versions_to_test_usb) }}
+        fail-fast: false
+      runs-on: ['self-hosted', 'testbed-runner']
+      steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)